### ISSUE TITLE
Question about data Split

### ISSUE THREAD

**Issue by U12458** [wklix/TransLNP#4]
Hello, great to see jobs related to the LNP field! When I reproduced your work, I noticed that there are two split methods: random and scaffold. But the training and testing data for two methods are pre-divided. It seems that you only use the random and scaffold data division for cross-validation on the training set, is that so?

**Comment by wklix** [comment]
Thanks for your question. I pre-divided the training set, test set, and validation set using both Random and Scaffold data splitting methods, and performed cross-validation on the training and validation sets.

---- Replied Message ----
| From | ***@***.***> |
| Date | 10/10/2024 11:55 |
| To | wklix/TransLNP ***@***.***> |
| Cc | Subscribed ***@***.***> |
| Subject | [wklix/TransLNP] Question about data Split (Issue #4) |

Hello, great to see jobs related to the LNP field! When I reproduced your work, I noticed that there are two split methods: random and scaffold. But the training and testing data for two methods are pre-divided. It seems that you only use the random and scaffold data division for cross-validation on the training set, is that so?
